### ISSUE TITLE
.asoundrc Needs to be Updated

### ISSUE THREAD

**Issue by AdamDanischewski** [Grippentech/Asus-E200HA-Linux-Post-Install-Script#15]
Hi, thanks for the script it works well. One small issue, I installed this yet sound wasn't working initially, it was providing buffer underruns. After playing around with it for a little while I figured out that plughw:1,0 was working. 

aplay /usr/share/sounds/alsa/Front_Center.wav   

Fails, out of the box after running the Post Install Script on Ubuntu 17.04. 

First checking, aplay -l for the devices with the new kernel after the scripts ran, 
I found that the proper default output device is Card 1 Device 0, you can verify that it works: 
aplay -Dplughw:1,0 /usr/share/sounds/alsa/Front_Center.wav  

The problem is that the ~/.asoundrc doesn't exist yet, so you have to create it and add the following to my ~/.asoundrc to get Alsa to use the appropriate device by default/automatically: 
```
pcm.!default { 
 type plug 
slave { 
  pcm "hw:1,0" 
} 
}
```
Thereafter, `aplay /usr/share/sounds/alsa/Front_Center.wav`, works now. 

Maybe you can add this to the installation script. 

+Adam Michael Danischewski

**Comment by kajetan-mazur** [comment]
Hi @AdamDanischewski your solution let me play sound from .wav but still didnt help me with youtube/my mute button. Still have my volume icon into mute :( https://github.com/Grippentech/Asus-E200HA-Linux-Post-Install-Script/issues/16

**Comment by AdamDanischewski** [comment]
I generally have the same problem that you do - I don't really need youtube to work for me, as long as VLC is working I'm fine. I'm using XFCE4 as the window manager. Grippentech on your thread seems to think that LXDE may make a difference have you tried that? It may be counter-intuitive yet the window manager in other alsa related threads mention that the window manager could be the issue.

**Comment by Grippy98** [comment]
Yes indeed. It's really bizarre but YouTube works fine under LXDE

**Comment by kajetan-mazur** [comment]
@AdamDanischewski i more worried about youtbue cause there are very interesting tutorials for programmers :)

**Comment by AdamDanischewski** [comment]
You can still get them, I looked into this issue a little bit more. It looks like there could be an issue with some competing modules and the pulse configuration. There may be a way to configure pulseaudio to fix this problem by hand + blacklisting, but I don't have time to figure it out at the moment. 

For now, I recommend you remove pulseaudio (sudo apt-get remove pulseaudio) and install youtube-dl that way you will have copies of the videos on your computer. Install vlc and use the first chtcx2072x, Direct sample mixing device. 

https://github.com/rg3/youtube-dl
